src/widgets/TableShortcuts.js

```diff
@@ -0,0 +1,83 @@
+/* eslint-disable react/forbid-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { View, Text, TouchableOpacity, StyleSheet } from 'react-native';
+
+import { APP_FONT_FAMILY, APP_GENERAL_FONT_SIZE } from '../globalStyles';
+
+export const TableShortcut = ({
+  children,
+  onPress,
+  shortcutKey,
+  containerStyle,
+  innerContainerStyle,
+  textStyle,
+}) => {
+  const Container = onPress ? TouchableOpacity : View;
+
+  const wrappedOnPress = React.useCallback(() => onPress(shortcutKey), []);
+  const renderChildren = React.useCallback(
+    child => (typeof child === 'string' ? <Text style={textStyle}>{child}</Text> : child),
+    []
+  );
+
+  return (
+    <View onPress={wrappedOnPress} style={containerStyle}>
+      <Container style={innerContainerStyle}>
+        {React.Children.map(children, renderChildren)}
+      </Container>
+    </View>
+  );
+};
+
+export const TableShortcuts = ({ children }) => {
+  const { shortcutsContainer } = localStyles;
+  return <View style={shortcutsContainer}>{children}</View>;
+};
+
+const localStyles = StyleSheet.create({
+  container: {},
+  shortcutContainer: {
+    flex: 1,
+    borderBottomColor: 'black',
```

Comment:
would be nice to have a global style for the border; wondered about using `SHADOW_BORDER` here.. perhaps a new color though, as the SHADOW_BORDER is pretty light

Reply:
added the shadow border as a default for now: ecdefa2 -- think basically all the styles might been to be passed through - possibly even no border might somehow look better..